extensions/amp-analytics/amp-analytics.md

```diff
@@ -99,15 +106,14 @@ is an https URL. These values may include placeholder tokens that can reference
 
 ```javascript
 "requests": {
-  "base": "https://example.com/analytics?a=${account}&u={$canonicalUrl}&t=${title}",
+  "base": "https://example.com/analytics?a=${account}&u=${canonicalUrl}&t=${title}",
   "pageview": "${base}&type=pageview",
   "event": "${base}&type=event&eventId=${eventId}"
 }
 ```
 
 ###### Vars
-The `vars` attribute specifies key-value pairs that can be referenced in `request` values. These are commonly used to
-specify publisher specific information.
+`amp-analytics` defines many basic variables that can be used in the requests. A list of all such variables is available [here](./analytics-vars.md). In addition, all the variables supported by [AMP HTML URL Variable Substitutions](../../spec/amp-var-substitutions.md) are also supported. The `vars` attribute in the configuration can be used to define new key-value pairs or override existing variables that can be referenced in `request` values. New variables are commonly used to specify publisher specific information.
```

Comment:
Small suggestion: break up into two paragraphs. Start second paragraph with 'The 'vars' attribute in the'.